list_updatable_packages

```diff
@@ -67,11 +74,41 @@ class Spec:
 
 
 def find_specs() -> Generator[Spec, None, None]:
+    if DEBUG: print('Finding plugin specs')
     for path in Path('packages').glob('*/*/*.spec'):
         spec = Spec(path)
         if spec.template in SUPPORTED_TEMPLATES and not spec.is_nightly:
             yield spec
 
+def parse_gemfile_lock(project: str) -> dict:
+    gemfile_lock = fetch_gemfile_lock(project)
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        with open(f'{tmpdirname}/Gemfile.lock', 'w') as fp:
+            fp.write(gemfile_lock)
+
+        with open(f'{tmpdirname}/Gemfile', 'w') as fp:
+            pass
+
+        shutil.copy('parse-gemfile-lock', f'{tmpdirname}/parse-gemfile-lock')
```

Comment:
Rather than copying it, should it instead find the location of `parse-gemfile-lock`? Like:
```python
script = Path(__file__).absolute().parent / 'parse-gemfile-lock'
```

Then call that in `check_output`

Reply:
Thanks, that was much cleaner, updated to use it.